packages/story-editor/src/components/panels/design/link/link.js

```diff
@@ -284,6 +327,40 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
           </Row>
         </>
       )}
+      {showRelOption && (
```

Comment:
Should these options be before the icon/optional description perhaps? Otherwise it can cause quite a lot of jumpiness during validation, e.g.
![jumpiness](https://user-images.githubusercontent.com/3294597/141762481-fa15d786-fd82-4eb9-866f-50c111111746.gif)

Reply:
Yeah, good point. I am pushed a fix.